src/styles/main.scss

```diff
@@ -0,0 +1,42 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
```

Comment:
It seems there is a typo in the variable name `$weakSize`. According to the common naming conventions, it should probably be `$weekSize`.